src/parser.js

```diff
@@ -22,13 +22,24 @@ define([
     }
 
     function getAttributes (element) {
-        var attributes = $(element)[0].attributes,
-            attrMap = {};
+        var attributes = _.chain($(element)[0].attributes)
+                .map(function (value) {
+                    return [value.nodeName, value.nodeValue];
+                }) .object().value(),
+            attrKeys = _.keys(attributes);
+
+        _.each(attrKeys, function (key) {
+            if (key.startsWith('vellum:') &&
```

Comment:
Turns out this is not needed at all https://github.com/dimagi/Vellum/commit/5e0b29373a68543695c7fbeef24089c6570caba6

It was useful when I was working on this piece by piece since this function would magically make things work, but since I implemented parseVellumAttrs this is not needed because all vellum attributes will be parsed there.